### DISCUSSION TITLE
Fingerprint test values changing with Github Action runs

### DISCUSSION THREAD

**Discussion by n-jay** [omnetpp/omnetpp#1091] [General]
Hi all,
recently I sent a [PR](https://github.com/inet-framework/inet/pull/929) to the INET repository with the inclusion of the LEACH routing protocol I'd developed.
As requested by the contributor guidelines, I did my best to write adequate tests.
While the fingerprint tests passed correctly in my local dev machine, they seem to change with each subsequent Github Action run, causing the build to fail. This is also inconsistent _between tests_ as well, causing some to pass during certain runs and others to fail.

I mentioned this as a [comment](https://github.com/inet-framework/inet/pull/929#issuecomment-1842608263) in the PR, but thought I'd ask as well since the code hasn't been reviewed yet.
Hope to get some feedback on this.

**Comment by rhornig** [comment]
Thanks. I have added my comments there.

**Comment by n-jay** [comment]
Acknowledged. Thanks @rhornig!
